Core/src/IfcGeometryConverter/GeomUtils.cpp

```diff
@@ -634,23 +634,32 @@ bool GeomUtils::isPointOnLineSegment( double& target_lambda,
 }
 
 /**********************************************************************************************/
-
-bool LineToLineIntersectionHelper(carve::geom::vector<2>& v1, 
-								  carve::geom::vector<2>& v2, 
-								  carve::geom::vector<2>& v3, 
-								  carve::geom::vector<2>& v4, double & r, double & s)
+/*! \brief Calculates distances between start points of the segment and their intersection. 
+* \param[in] initialPointSeg1		Initial point of the first intersecting segment.			
+* \param[in] terminalPointSeg1		Terminal point of the first intersecting segment.
+* \param[in] initialPointSeg2		Initial point of the second intersecting segment.			
+* \param[in] terminalPointSeg2		Terminal point of the second intersecting segment.
+* \param[out] distToIntesection1	Distance from initialPointSeg1 to the segment intresection, which is calculated as section of the first segment. 
+* \param[out] distToIntesection2	Distance from initialPointSeg2 to the segment intresection, which is calculated as section of the second segment. 
+*/
+bool LineToLineIntersectionHelper(const carve::geom::vector<2>& initialPointSeg1,
+								  const carve::geom::vector<2>& terminalPointSeg1,
+								  const carve::geom::vector<2>& initialPointSeg2,
+								  const carve::geom::vector<2>& terminalPointSeg2, 
+	double & distToIntesection1, double & distToIntesection2)

```

Comment:
please make sure, that the indentation is consistent.

Reply:
Resolved: 8f80eb1